deploy/osd-cluster-ready/60-osd-ready.Job.yaml

```diff
@@ -11,8 +11,14 @@ spec:
             name: osd-cluster-ready
         spec:
             containers:
-            - name: osd-cluster-ready
+            # TODO: Remove this override once
+            # https://bugzilla.redhat.com/show_bug.cgi?id=1921413
+            # is resolved
+            - env:
```

Comment:
Do we need to bump the `backoffLimit` here as well in order to accommodate retries? (I can't actually make a comment on that line because it wasn't changed in the PR)

Reply:
Oh, yes, probably. And the `activeDeadlineSeconds`. IIUC, the Job will quit on whichever one it hits first.

I'm thinking we should just remove these and let the defaults play.

Reply:
> separate PR

#663 and https://github.com/iamkirkbater/osd-cluster-ready-job/pull/2